source/models/R290-generic.yaml

```diff
@@ -916,6 +1018,123 @@ add:
       min_value: 0
       max_value: 120
       step: 1
+    # Register 280
+    - platform: modbus_controller
+      modbus_controller_id: "${devicename}"
+      name: "Zone 2 T1SetC1 Custom Curve Cooling"
+      id: "${devicename}_zone2_t1setc1"
```

Comment:
The id value is different then the name, please make the id exact as the name, where spaces are changed to underscores and all letters are lowercase.

There are multiple instances of this in this change request. Please fix them all.